utils/umaApi.ts

```diff
@@ -127,6 +121,8 @@ export type Synth<T extends { type: ContractType }> = ContentfulSynth &
   SynthStats &
   SynthState<T> & { tvl24hChange: number };
 
+export type AnySynth = Synth<EmpState | LspState>;
```

Comment:
so that's how you do a generic synth!! This will be super useful, thx for adding that in